chainerx_cc/chainerx/array_repr.cc

```diff
@@ -106,7 +107,7 @@ class FloatFormatter {
             }
             os << std::scientific << std::left << std::setw(width) << std::setprecision(8) << value;
         } else {
-            if (std::isinf(value) || std::isnan(value)) {
+            if (std::isinf(value) || IsNan(value)) {
```

Comment:
Should we use `IsInf` too?

Reply:
Fixed it
https://github.com/durswd/chainer/commit/817bafdd8d93433c1c29feda470f87e9d5b19e81